### DISCUSSION TITLE
samples/rtr/bulk_execute.py truncates commands output stdout

### DISCUSSION THREAD

**Discussion by TyraelTLK** [CrowdStrike/falconpy#991] [Q&A] [answered]
Where I can find how RTR runscript raw truncates command output? 
Trying samples/rtr/bulk_execute.py I have noticed that the commands stdout is not complete. 
Is it possible to get the full stdout?

**Comment by jshcodes** [comment, marked as answer]
This sample needs to be updated to handle outputs that exceed a single result.  

> This is done by checking the return value __`complete`__. When this value is __False__, the next page of output can be retrieved by performing the [RTR_CheckCommandStatus](https://www.falconpy.io/Service-Collections/Real-Time-Response.html#rtr_checkcommandstatus) operation using the returned _`task_id`_ and incrementing the `sequence_id` keyword.

Creating an issue for this, thank you for pointing it out!

**Reply by TyraelTLK** [reply]
Thank you, this is the issue right?
https://github.com/CrowdStrike/falconpy/issues/992

**Reply by jshcodes** [reply]
Correct!

Can we see an example of the RTR command you're attempting to execute with this sample?

**Reply by TyraelTLK** [reply]
The first time I had the issue was with:
`vboxmanage list extpacks`
